server/src/main/java/com/android/example/KeyAttestationExample.java

```diff
@@ -310,28 +317,30 @@ public static void main(String[] args) throws Exception {
     ASN1Set purposes = (ASN1Set) findAuthorizationListEntry(teeEnforced, KM_TAG_PURPOSE);
     System.out.println("Key purpose(s): " + purposes);
     if (purposes == null
-        || purposes.toArray().length != 1
+        // || purposes.toArray().length != 1
```

Comment:
Why is this line commented out?

Reply:
The length has changed to 2, but since the plan is just to print out whatever's being used instead of checking what's expected, it doesn't matter at this point whether we check for ```lenght != 2``` or whether it's just commented out.